README.md

```diff
@@ -1,66 +1,104 @@
-# Wasmtime: a WebAssembly Runtime
+<div align="center">
+  <h1><code>wasmtime</code></h1>
 
-**A [Bytecode Alliance][BA] project**
+  <strong>A <a href="https://bytecodealliance.org/">Bytecode Alliance</a> project</strong>
 
-Wasmtime is a standalone wasm-only optimizing runtime for [WebAssembly] and
-[WASI]. It runs WebAssembly code [outside of the Web], and can be used both
-as a command-line utility or as a library embedded in a larger application.
+  <p>
+    <strong>A standalone runtime for
+    <a href="https://webassembly.org/">WebAssembly</a></strong>
+  </p>
```

Comment:
I wonder if this should be above the "a bytecode alliance project" bit? I would sort of expect that at the bottom of the header, like "made with <3 by the bytecode alliance" or something like that.

Reply:
This is currently following the idioms we have in other projects like https://github.com/bytecodealliance/wat, but I'd be fine swapping the order here myself